components/sensor/bmp3xx.rst

```diff
@@ -0,0 +1,94 @@
+BMP388 / BMP390 Temperature+Pressure Sensor
+===========================================
+
+.. seo::
+    :description: Instructions for setting up BMP388 or BMP390 temperature and pressure sensors with ESPHome
+    :image: bmp388.jpg
+    :keywords: BMP388 BMP390
+
+The ``bmp3xx`` sensor platform allows you to use your BMP388 or BMP390 
+(`datasheet <https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp390-ds002.pdf>`__, `BMP390 product page <https://www.bosch-sensortec.com/products/environmental-sensors/pressure-sensors/bmp390/>`__) temperature and pressure sensors with ESPHome. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/bmp388.jpg
+    :align: center
+    :width: 50.0%
+
+    BMP388/BMP390 Temperature and Pressure Sensor.
+
+.. _Adafruit: https://www.adafruit.com/product/2651
```

Comment:
Stray link.

Reply:
Thanks for the improvements - implemented